vespa-hadoop/pom.xml

```diff
@@ -17,109 +17,28 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hadoop.version>3.3.4</hadoop.version>
-        <woodstox.version>6.4.0</woodstox.version>
-        <sun-jersey-json.version>1.19.4</sun-jersey-json.version>
-        <jettison.version>1.5.1</jettison.version>
-        <pig.version>0.16.0</pig.version>
-        <tomcat-jasper.version>5.5.23</tomcat-jasper.version>
     </properties>
 
-    <dependencyManagement>
-        <dependencies>
-            <!-- Force newer version of jetty-util (to match Jetty version of jdisc) -->
-            <dependency>
```

Comment:
FYI the unit test framework for hadoop/pig requires Jetty 9.4. We'll have to reintroduce dependency management to force 9.4 once we migrate to Jetty 11.

Reply:
I noticed, better to start off from scratch.